README.md

```diff
@@ -25,3 +25,6 @@
 
 `/ Nikoll Daiana Quintero Chavez`
 8. Etiqueta sectionFive 
+
+`/ Sofía Miranda Galvis`
+8. Etiqueta list (horizontal y vertical) 
```

Comment:
Change the number eight to nine

Reply:
Yes, I noticed that error, and it was corrected later on.